_sass/components/_header.scss

```diff
@@ -0,0 +1,297 @@
+#skiplink {
+  position: absolute;
+  top: 0;
+  left: 0;
+  z-index: 10000;
+  width: 250px;
+  left: 50%;
+  margin-left: -125px;
+  padding: 10px;
+  background: white;
+  text-align: center;
+  border: 1px solid black;
+  color: black;
+  display: block;
+}
+
+header {
+  background: $vanilla url('../img/sdgbg.png') bottom left repeat-x;
+  .container {
+    min-height: 113px;
+    .navbar-default {
+      border: none;
+    }
+  }
+}
+
+.navbar-brand {
+  display: block;
+  width: 75%;
+  max-width: 300px;
+  img {
+    width: 100%;
+  }
+}
+
+.navbar {
+
+  margin-bottom: 32px;
+
+  li {
+    text-transform: uppercase;
+  }
+
+  ul.navbar-nav {
+    float: right;
+    margin-top: 12px;
+
+    li {
+      a {
+        padding: 2px;
+        margin: 2px 15px;
+        color: #595959;
+      }
+
+      &.active a {
+        border-bottom: 2px solid #333;
+        background: transparent;
+        color: #333;
+
+        &:hover {
+          background: transparent;
+        }
+      }
+
+      a:last-child {
+        margin-right: 0;
+      }
+
+      &.contrast {
+        @media only screen and (max-width: 768px) {
+          display: none;
+        }
+        font-size: 1.3em;
+        font-weight: bold;
+        border-radius: 50%;
+        width: 25px;
+        max-height: 25px;
+        text-align: center;
+        margin-left: 0.5em;
+        a {
+          margin: auto;
+          text-decoration: none;
+        }
+      }
+
+      &.contrast-default {
+        background: white;
+        a {
+          color: black;
+        }
+      }
+
+      &.contrast-high {
+        background: black;
+        a {
+          color: yellow;
+        }
+      }
+    }
+  }
+
+  &.navbar-default {
+    background: transparent;
+    border: none;
+    float: right;
+  }
+
+  .top-level {
+    display: none;
+  }
+
+  #search {
+    position: relative;
+    clear: both;
+    float: right;
+
+    label {
+      position: absolute;
+      top: 5px;
+      right: 0px;
+      font-size: 1.40em;
+
+      span {
+        display: none;
+      }
+    }
+
+    input {
+      padding: 5px;
+      padding-left: 5px;
+      width: auto;
+      height: 34px;
+      transition: width 0.4s;
+      margin-top: 5px;
+
+      &:focus {
+        width: 350px;
+      }
+    }
+  }
+}
+#search-btn {
+  height: 34px;
+  }
+
+.table {
+  .table-value {
+    text-align: right;
+  }
+}
+
+@media only screen and (max-width: 768px) {
```

Comment:
To clean this up even further, you could split your files to reflect different responsive layouts.

Reply:
Do you mean to have, for example, a `_header__desktop.scss` file, `_header__mobile.scss`, etc? I agree this would clean things up. But I admit I'm a little hesitant to launch into that. I don't have a good handle on all the media queries, and whether we're consistent with "mobile first" or "desktop first" in general.

Reply:
Fair enough. We can leave it as is if it's too much work for now. We can back to it on a later date if needed.